xls/netlist/cell_library_test.cc

```diff
@@ -138,6 +139,29 @@ TEST(CellLibraryTest, EvaluateStateTable) {
       table.GetSignalValue(stimulus, "ham_sandwich"),
       status_testing::StatusIs(absl::StatusCode::kNotFound,
                                ::testing::HasSubstr("No matching row")));
+
+  EXPECT_THAT(table.GetSignalValue(stimulus, "PB&J"),
+              status_testing::StatusIs(absl::StatusCode::kInvalidArgument));
+}
+
+TEST(CellLibraryTest, LutStateTable) {
+  // 4-way AND
+  StateTable table = StateTable::FromLutMask(0x8000);
+  StateTable::InputStimulus stimulus;
+  stimulus["I0"] = true;
+  stimulus["I1"] = true;
+  stimulus["I2"] = true;
+  stimulus["I3"] = true;
+  XLS_ASSERT_OK_AND_ASSIGN(bool signal, table.GetSignalValue(stimulus, "X"));
```

Comment:
You can do:

EXPECT_THAT(table.GetSignalValue(stimulus, "X"), IsOkAndHolds(true));

same below.

Reply:
fancy!